go.tools.mod

```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	github.com/jstemmer/go-junit-report v0.9.1 // indirect
-	github.com/mattermost/mattermost-utilities/mmgotool v0.0.0-20200915114419-f4421bc07461 // indirect
+	github.com/mattermost/mattermost-utilities/mmgotool v0.0.0-20200926180007-fd1b679200e5 // indirect
```

Comment:
Unrelated, but I am recently seeing this. `make i18n-extract` will automatically bump the version in go.tools.mod, even when there is no `-u`. Do you know why this would happen? I need to see.

Reply:
Not sure. I just ran `make i18n-extract` to update some missing translations and the version got bumped. I can revert this.